accounts/addproofer.php

```diff
@@ -263,7 +270,7 @@ function _validate_fields($real_name, $username, $userpass, $userpass2, $email,
     // forum software which, if used, will cause account creation to fail in
     // activate.php.
 
-    if(!$referrer)
+    if(!$referrer || ($referrer == 'other' && !$referrer_explanation))
```

Comment:
Checking that `$referrer_explanation` isn't empty makes sense but we should run it past Linda. It's one thing to give the user an option to fill out `other` and another to require them to do so before continuing. Consider pinging her in the #pgdpdev channel.

Reply:
switched it to referrer_details per @srjfoo.